charts/cloudzero-agent/README.md

```diff
@@ -64,6 +64,54 @@ prometheus-node-exporter:
 ```
 This will deploy the required resources for metric scraping.
 
+### Custom Scrape Configs
+If the chart is running *without* the `kube-state-metrics` and `prometheus-node-exporter` exporters enabled (meaning, those two exporters are deployed from some other source outside of this chart), then the scrape configs used by the underlying Prometheus agent may need to be adjusted.
+
+As an example, the out-of-the-box scrape config in this chart attempts to find the `kube-state-metrics` exporter via an annotation on a k8s `endpoints` resource deployed by the KSM subchart. If `kube-state-metrics` was instead deployed without any annotations, and was only available via a Service, we could add the following:
+
+custom-scrape-config.yaml
+```yaml
+prometheusConfig:
+  scrapeJobs:
+    kubeStateMetrics: # this disables the default kube-state-metrics scrape job, which will be replaced by an entry in additionalScrapeJobs
+      enabled: false
+    additionalScrapeJobs:
+    - job_name: custom-kube-state-metrics
+      honor_labels: true
+      honor_timestamps: true
+      scrape_interval: 1m
+      scrape_timeout: 10s
+      metrics_path: /metrics
+      static_configs:
+        - targets:
+          - 'my-kube-state-metrics-service.default.svc.cluster.local:8080'
```

Comment:
So we want to call attention to the use of the DNS address here is a side note below?